src/Runtime/workflow-engine/WORKFLOW_COLLECTIONS.md

```
@@ -0,0 +1,324 @@
+Batch 1:  POST /api/v1/workflows
+          CollectionKey = "daily-sync"
+          Workflows: [A, B]    (no deps between them)
+
+          Collection state after:
+          Heads = [A, B]       (both are leaves — nothing depends on them)
+
+              A       B
+              ●       ●  ← heads
+```
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)